kv/txn_coord_sender_test.go

```diff
@@ -195,22 +150,17 @@ func TestTxnCoordSenderBeginTransactionMinPriority(t *testing.T) {
 	defer s.Stop()
```

Comment:
I'm not sure I understand what this test does, at least from the comment. You got an idea?